app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -17,8 +17,6 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     end
 
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
-    api :GET, "/systems/:system_id/subscriptions", N_("List a content host's subscriptions"), :deprecated => true
-    api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
```

Comment:
This would seem to be moving away from the pattern that we have been converging towards which is to use the index method of a resource to handle these types of queries. Is there a technical reason to make an exception here?